pkg/etcd/etcd.go

```diff
@@ -198,6 +198,11 @@ func (e *ETCD) Reset(ctx context.Context, rebootstrap func() error) error {
 		t := time.NewTicker(5 * time.Second)
 		defer t.Stop()
 		for range t.C {
+			// resetting the apiaddresses to nil since we are doing a restoration
+			if _, err := e.client.Put(ctx, AddressKey, ""); err != nil {
```

Comment:
I was going to ask why Put empty string instead of just Delete but i see that our Put is much easier to use since it handles getting the current revision for you...

Reply:
yeah exactly :D